shared/src/commonMain/kotlin/co/nimblehq/ic/kmm/suv/data/model/ApiToken.kt

```diff
@@ -0,0 +1,47 @@
+package co.nimblehq.ic.kmm.suv.data.model
+
+import co.nimblehq.ic.kmm.suv.domain.model.Token
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+// TODO: Find a JsonApi to support this
+
+@Serializable
+data class ApiToken(
```

Comment:
Bring this from iOS 🙈. We will have different data sources (e.g., Remote/API, Local, Realm, ...). So it's better to have `type of source` in the name. However, when checking [this section](https://developer.android.com/topic/architecture/data-layer#business-models). What do you think if I change to `TokenApiModel` (same as UiModel 🤔)? In case we have another type of source (Realm, SQLite,...), we will have `TokenRealmModel` or `TokenSQLiteModel`. What do you think?

Reply:
@suho If we go with `TokenApiModel`, I'm wondering how do we name the request body model? How about `TokenApiResponse` and `LoginApiRequest` 💭

Reply:
I would not refer to use `Response` and `Request`. Technically, With the suffix `Response`, it's a response with more information (status code, header, and its body).

So I prefer to use `TokenApiModel` for the response model, and for the request body, we could use `LoginApiRequestBody` or `LoginApiBody` for shorter and clearer? 🤔

Reply:
Okay, let's with your updates @suho 👍